WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -334,27 +334,27 @@ private void createNotificationChannelsOnSdk26() {
             NotificationChannel normalChannel = new NotificationChannel(
                     getString(R.string.notification_channel_normal_id),
                     getString(R.string.notification_channel_general_title), NotificationManager.IMPORTANCE_DEFAULT);
-            // Register the channel with the system; you can't change the importance
-            // or other notification behaviors after this
+            // Register the channel with the system
```

Comment:
This also changed the message. 

- _"you can't change the importance"_: Assertive and says that we *cannot*
- _"Do not change the importance"_: Says we *can* but don't do it. 

The lines below these have the same issues.

Can we revert these back? 🙂